inspirehep/modules/search/walkers/elasticsearch.py

```diff
@@ -90,6 +90,12 @@ def visit(self, node):
     @visitor(Value)
     def visit(self, node):
         def query(keyword):
+
+            # FIXME: This is a temporary hack that should be removed when
+            # nested keywords search is implemented
+            if str(node.value).startswith('recid:'):
+                node.value = node.value[len('recid:'):]
```

Comment:
Were you as surprised as me to see that there's no method in the standard library for this? http://stackoverflow.com/a/16891418/374865